pages/contact/index.tsx

```diff
@@ -6,15 +6,15 @@ const ContactUs: React.FC = () => {
   return (
     <>
       <Flex margin="auto" maxWidth="80%" marginTop="50px" direction="column">
-        <Heading>We would love to hear from you and keep you updated with our progress!</Heading>
+        <Heading>We would love to hear from you and keep you updated on our progress!</Heading>
         <Box margin="50px 0">
           <Box margin="25px 0">
             <Text>
-              For questions, press inquiries, or other issues please contact us at{' '}
+              Please contact us at{' '}
```

Comment:
I think it would make sense to make a string template. {`please contact us at `}